app_dart/lib/src/request_handlers/check_for_waiting_pull_requests.dart

```diff
@@ -284,7 +284,7 @@ class CheckForWaitingPullRequests extends ApiRequestHandler<Body> {
         allSuccess = false;
```

Comment:
`CheckRun.fromJson(checkRun)` would be clearer than using the `Map<String, dynamic>`

Reply:
I'd rather we change that in a separate commit in case it breaks anything.